### ISSUE TITLE
Question about teacher model performance in the paper

### ISSUE THREAD

**Issue by ofishhaha** [pppppM/mmsegmentation-distiller#2]
Hi,

I happen to found that the results of PSPnet-r101 on Pascal VOC（mIoU=78.52, mAcc=79.57）is exactly the same as the official model provided by mmsegmentation.

1、Have you tested the pretrained model? or directly use the accuracy on the webpage? or train it independently?
I have this question because I also tested the pretrained model and haven't got the same result. I will appreciate it if you can explain it a little.

2、Maybe it has an error in the mAcc result. because in the mmsegmentation webpage, 79.57 is reported as mIoU(ms+flip) not mAcc. After my test, the mAcc is about 84.22% for the teacher model. Is it my misunderstanding?

Thank you for your efforts.

**Comment by irfanICMLL** [comment]
Sorry for the typo. The weight of the teacher model is the mmseg trained one, thus, the test results should be exactly the same as the webpage.
